### DISCUSSION TITLE
Face detections and clusters in external folders mounted for multiple users

### DISCUSSION THREAD

**Discussion by dennisverspuij** [nextcloud/recognize#1147] [Q&A]
Hi, I am struggling to have face detection and clustering working for multiple users that have the same external storage folder mounted.
To be more precise, at first I have this external storage folder X mounted for user A, and have succesfully detected and clustered faces for this user. Later I added access and mounted X for user B as well, but no people appear. So I called `occ recognize:recrawl` and I see the queues filled up for scanning the files again, and see the number dropping to 0 as being processed. But still no people appear for user B. When I peek into the oc_recognize_face_detections table, no entries for user B were added,, only those for user A still exist. Similar case in oc_recognize_face_clusters table. What am I doing wrong here? I am aware of multiple issues regarding sharing face detections and clustering,, such as [mentioned and referenced here](https://github.com/nextcloud/recognize/discussions/1063). But these don't hint at the issue I am having. Any help much appreciated!

**Comment by marcelklehr** [comment]
Hi @dennisverspuij 
Which version are you using? I think this warrants creating a proper issue instead of this discussion, could you take care of this and fill out the whole questionnaire? :blue_heart:

**Reply by dennisverspuij** [reply]
Please see issue [1148](https://github.com/nextcloud/recognize/issues/1148)
